src/popover/popover.ts

```diff
@@ -150,13 +147,26 @@ export class NgbPopover implements OnInit, OnDestroy {
     });
   }
 
+  /**
+   * Content to be displayed as popover. If falsy, the popover won't open.
+   */
+  @Input()
+  set ngbPopover(value: string | TemplateRef<any>) {
+    this._ngbPopover = value;
+    if (!value && this._windowRef) {
```

Comment:
This check needs to be more strict - what if I want to open a popover with a single `0` value?

Reply:
Good point. There is no problem if the user passes value as string `[ngbPopover]="!shouldDisplayPopover ? '0' : null"`